com.endlessm.CompanionAppService.json.in

```diff
@@ -39,6 +39,86 @@
                 }
             ]
         },
+        {
+            "name": "python-dbus",
+            "cleanup": ["*"],
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links \"file://${PWD}\" --prefix=${FLATPAK_DEST} dbus-python"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://pypi.python.org/packages/ad/1b/76adc363212c642cabbf9329457a918308c0b9b5d38ce04d541a67255174/dbus-python-1.2.4.tar.gz",
+                    "sha256": "e2f1d6871f74fba23652e51d10873e54f71adab0525833c19bad9e99b1b2f9cc"
+                }
+            ]
+        },
+        {
+            "name": "python-dbusmock",
+            "cleanup": ["*"],
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links \"file://${PWD}\" --prefix=${FLATPAK_DEST} python-dbusmock"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://pypi.python.org/packages/4d/cd/a5f571ef37addb45a86eca2e66a6b4789d915dbb9170a997dba02b7a0a89/python-dbusmock-0.17.tar.gz",
+                    "sha256": "58f802ef4c659a1adbca31eb6c26688f1c09599b5832354eb258340b66673346"
+                }
+            ]
+        },
```

Comment:
Quite surprising that we need these two - what did we need them for?

Reply:
In any event, can these be generated into `com.endlessm.PipDependencies.json` by adding them to `requirements.txt` and then using `make regenerate-pip-manifest-template`?

Reply:
They're needed as dependencies for eos-metrics. See https://phabricator.endlessm.com/T22554
I prefer not to put them in the `requirements.txt` of `eos-companion-app-integration` since they are not dependencies of that project.

Reply:
Ah, fun. Okay, lets proceed with this then - I built it locally and it worked fine.